src/main/java/de/skuzzle/enforcer/restrictimports/impl/ImportMatcherImpl.java

```diff
@@ -107,7 +107,12 @@ private static String extractPackageName(String line) {
     }
 
     private static boolean is(String compare, String line) {
-        return line.startsWith(compare) && line.endsWith(";");
+        String stripped = line.replaceAll("\\/\\*.*?\\*\\/", "");
```

Comment:
Could you extract this into a pre-compiled Regex using `Pattern.compile("....")`? Should be way faster because this line is executed very often

Reply:
Sure